tests/monitoring_test.go

```diff
@@ -75,6 +105,63 @@ var _ = Describe("[Destructive] Monitoring Tests", func() {
 			err = f.K8sClient.StorageV1().StorageClasses().Delete(context.TODO(), name, metav1.DeleteOptions{})
 			Expect(err).ToNot(HaveOccurred())
 		}
+	}
+
+	createStubSnapshotClass := func(driver string) {
+		err := f.CrClient.Create(context.TODO(), newStubSnapshotClass(driver))
+		Expect(err).ToNot(HaveOccurred())
+		f.SnapshotSCName = defaultStorageClass.Name
```

Comment:
I think it's a potential foot gun to change these framework values after init

Reply:
Agree, will fix. Wanted it for the `IsSnapshotStorageClassAvailable()` check, but I can just add `!isStubSnapshotClass` to following `Skip()` condition instead.